types/office-js-preview/index.d.ts

```diff
@@ -37651,22 +37682,6 @@ declare namespace Excel {
     class ChartFill extends OfficeExtension.ClientObject {
         /** The request context associated with the object. This connects the add-in's process to the Office host application's process. */
         context: RequestContext;
-        /**
```

Comment:
@wandyezj This block is removed from office-js-preview in the most recent CDN update, can you confirm it should be removed?

Reply:
It's ok for beta APIs to be removed from the office-js-preview.